App.js

```diff
@@ -38,7 +42,7 @@ export default class App extends Component {
     this.sdkCrashEventSubscription.remove();
```

Comment:
@countcain When `setupSdk` does not complete and we exit `componentDidMount` via the `catch` block, this line here, along with the preceding two, will result in an error due to these variables not having been initialized (`subscribeAfterSDKSetup` will never have been called).

Reply:
Yes. the app will crash here.